tests/integration_test_net.go

```diff
@@ -65,7 +65,15 @@ func StartIntegrationTestNet(directory string) (*IntegrationTestNet, error) {
 	// initialize the data directory for the single node on the test network
 	// equivalent to running `sonictool --datadir <dataDir> genesis fake 1`
 	originalArgs := os.Args
-	os.Args = []string{"sonictool", "--datadir", result.stateDir(), "genesis", "fake", "1"}
+	os.Args = []string{
+		"sonictool",
+		"--datadir", result.stateDir(),
+		"genesis",
+		"fake",
+		"--statedb.livecache", "1",
+		"--statedb.archivecache", "1",
```

Comment:
same here, 1 byte? is the scale correct?

Reply:
There is not any reason in particular, I just saw we use this value already in tests. Both 0 and 1 sets lowest value in Carmen which is 2 KB